system/jlib/jstatcodes.h

```diff
@@ -66,7 +68,7 @@ enum StatisticScopeType
     SSTactivity,
     SSTallocator,                       // identifies an allocator
     SSTsection,                         // A section within the query - not a great differentiator
-    SSTcompilestage,                    // a stage within the compilation process
+    SSToperation,                       // an operation or stage in processing
```

Comment:
A good point.  I started off using >compile everywhere, and then relaxed it, but you are correct it will be inconsistent, which is not great.  It could be also be a SSTstage where it only matches specific words with no prefix - if so would there be any other examples it could be used for?  Your "dfu" scope could be another.

Reply:
SSTstage sounds like a good idea.  I agree it could be used for dfu as well.  I assume there would be a prefix for stage?